Drift/Kernel.php

```diff
@@ -68,7 +70,12 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
     {
         $confDir = $this->getApplicationLayerDir().'/config';
         $container->setParameter('container.dumper.inline_class_loader', true);
-        $loader->load($confDir.'/services.yml');
+        $container->setParameter('container.dumper.inline_factories', true);
+
+        $loader->load($confDir.'/{packages}/*'.self::CONFIG_EXTS, 'glob');
+        $loader->load($confDir.'/{packages}/'.$this->environment.'/*'.self::CONFIG_EXTS, 'glob');
+        $loader->load($confDir.'/{services}'.self::CONFIG_EXTS, 'glob');
+        $loader->load($confDir.'/{services}_'.$this->environment.self::CONFIG_EXTS, 'glob');
```

Comment:
Let me start a discussion here.

Symfony turned too magical in this part, and this code block opens the door to **many** different ways to work in the same project with the same result. This is something I'd like to avoid.